routes/carts.js

```diff
@@ -0,0 +1,80 @@
+const express = require('express');
+const routerCarts = express.Router();
+const knex = require('../database');
+
+routerCarts.route('/:user_id').get((req, res) => {
+  const id = req.url.slice(1);
+  knex
+    .raw('SELECT id FROM users WHERE id = ?', [id])
+    .then((result) => {
+      if (result.rows.length === 0) {
+        return res.send(404, '{ "message": "User not found" }');
+      }
+      knex
```

Comment:
Return this promise or else any error that are generated from within this promise are lost.